### ISSUE TITLE
cross/multi container dnd using useSortable with a single context

### ISSUE THREAD

**Issue by FalconiZzare** [clauderic/dnd-kit#1391]
# 🐛 Need Help With Setting Up Cross/Multi Container DnD
## 📝 Description Of The Issue
I'm using DnDContext, 1 SortableContext inside the main container where Category Containers can be Dragged and Dropped and category containers have SortableContext inside each of them to drag and drop subcategories. All of these using a single DnDContext. Now I want to move subcategories cross-container, basically I want to achieve multi container drag and drop for the subcategories. That's where the issue begins, broken drag and drop animations/visuals. I can't set it up properly. If anyone can help me set it up properly, at least the drag and drop visuals/animations where it doesn't break down while moving cateogory containers around and moving one subcategory to another category container. I prefer the constants the way they are structred. Will be beneficial for me.
## 👀 Visual Preview Of The Issue
https://github.com/clauderic/dnd-kit/assets/65172465/17d53f10-8cda-448d-8171-208c8871b455
## 🔗Link to repositories
GitHub: [Link](https://github.com/FalconiZzare/dnd-issue)

GitHub Pages: [Link](https://falconizzare.github.io/dnd-issue/)
## ⚙️ My Environment
| Package  | Version |
| ------------- | ------------- |
| node  | v21.7.1  |
| react  | v18.2.0  |
| @dnd-kit/core | v6.1.0 |
| @dnd-kit/modifiers | v7.0.0 |
| @dnd-kit/sortable | v8.0.0 |
| @dnd-kit/utilities | v3.2.2 |

**Comment by musiur** [comment]
Yes, I'm looking for the same thing. In native HTML5 we do have multi container facility.

**Comment by FalconiZzare** [comment]
@musiur I think they fixed this with the newer version. The new version has more feature, is easy to setup and more example now.

**Comment by clauderic** [comment]
This issue was opened against a previous version of dnd-kit. The library has since undergone a major rewrite.

It has been rebuilt from the ground up with support for **multiple frameworks** (React, Vue, Svelte, Solid, and vanilla JavaScript), improved performance, and a new API.

- **Documentation**: https://dndkit.com
- **Migration guide**: https://dndkit.com/react/migration
- **GitHub**: https://github.com/clauderic/dnd-kit

If this issue is still relevant to the latest version, please open a new issue.
